### PR TITLE
Hide and show graph MA-110

### PR DESCRIPTION
**Pull request by Miles-Alford**
WHAT: Added hide and show graph on case details page.

WHY: So that the user can tailor the case detail page to suit their preference.

Screenshot 2019-09-23 at 15 07 21

### CODE REVIEW

**Comment by georgeschena** [comment]
Could you add a screenshot of the UI please!

File: spec/features/view_a_case_spec.rb [hunk 1]
```diff
@@ -50,9 +50,8 @@ def then_i_should_see_case_account_balance
 
   def then_i_should_see_balance_graph
     expect(page.body).to have_css('h2', text: 'Balance over time', count: 1)
+    expect(page.body).to have_css('span', text: 'Graph', count: 1)
```
Comment (kjdchapman) [inline spec/features/view_a_case_spec.rb]: Does this produce a false positive? I'm not sure if this component being hidden will stop capybara finding it.

File: app/views/tenancies/show.html.erb [hunk 2]
```diff
@@ -73,13 +73,20 @@
...
+      <details class="govuk-details" data-module="govuk-details">
+      <summary style="color:#1d70b8; cursor:pointer" class="govuk-details__summary">
+    <span class="govuk-details__summary-text"><u>Graph</u></span>
+  </summary>
+    <canvas id="balance_chart"></canvas> 
+    </details>
   </div>
 </div>
 
+
```
Comment (kjdchapman) [inline app/views/tenancies/show.html.erb]: Can lose these extra empty lines?

File: app/views/tenancies/show.html.erb [hunk 3]
```diff
@@ -195,6 +202,7 @@
       var transactions = <%= from_last_year_as_json(@tenancy.transactions) %>
         window.arrearsGraph && window.arrearsGraph.destroy();
       window.arrearsGraph = BalanceGraph(ctx, transactions)
+      ctx.style.height = "300px"
```
Comment (kjdchapman) [inline app/views/tenancies/show.html.erb]: What does this line do?

I'm worried that a single custom style line in the markup file will make this hard to maintain in future, especially if we're trying to make these pages work on tablets at some point.

Comment (elena-vi) [inline app/views/tenancies/show.html.erb]: is there any reason why we're specifying this height here?

Comment (Miles-Alford) [reply app/views/tenancies/show.html.erb]: if we don't we get an issue where the graph is drawn with a starting height of 50px.

File: spec/features/view_a_case_spec.rb (hunk 1 above)
Comment (maysakanoni) [reply spec/features/view_a_case_spec.rb]: I think the span element is always there which is what they're testing for.

File: app/views/tenancies/show.html.erb (hunk 3 above)
Comment (Miles-Alford) [reply app/views/tenancies/show.html.erb]: If its style is not set when it is draw, the graph starts with a height of 50px. squashing the graph similar to the issue we had when using javascript to show the graph

File: spec/features/view_a_case_spec.rb (hunk 1 above)
Comment (kjdchapman) [reply spec/features/view_a_case_spec.rb]: We talked a little around this irl about the trade-offs between introducing javascript testing to cover this feature, I was happy with testing the js-less version (where the graph will still be visible).

**Pull request merged**